packages/wonder-stuff-core/src/__tests__/kind-error.test.js

```diff
@@ -240,6 +273,30 @@ describe("KindError", () => {
                 // Assert
                 expect(error.message).toBe(combinedErrorInfo.message);
             });
+
+            it("should have a combined message that cascades all causes", () => {
+                // Arrange
+                const cause1 = new Error("CAUSE_MESSAGE");
+                const cause2 = new KindError(
+                    "CAUSE_2_MESSAGE",
+                    Errors.Unknown,
+                    {cause: cause1},
+                );
+
+                // Act
+                const result = new KindError("ROOT_MESSAGE", Errors.Unknown, {
+                    cause: cause2,
+                });
+
+                // Assert
+                expect(result.message).toMatchInlineSnapshot(`
+                    "UnknownError: ROOT_MESSAGE
```

Comment:
Though, they are in different font-sizes/boldnesses, so it's not _quite_ so awkward...

Reply:
The webapp code could have originally been like this, but then I changed it on mobile because I noticed that Sentry was printing the error-kind twice.

Reply:
I'm also not super attached to this either, if it's easier for you to leave as-is!

Reply:
I think it needs changing, like you suggest - I'm going to do it in an upcoming PR rather than this one.